src/how-to-do-links-to-other-pages.wiki

```diff
@@ -1,28 +1,24 @@
 =How to do links to other pages?=
 
-Examples:
+If you want to make a link to an other page in your code, you can use the Eliom_content.Html.D.a function which is the functional representation for the <a> HTML tag. Supposing Eliom_content.Html.D is opened, here some examples with the previous created services:
```

Comment:
Please use monospace where appropriate, e.g., `{{{Eliom_content.Html.D.a}}}` and `{{{<a>}}}`.